### DISCUSSION TITLE
Smart notification vibration

### DISCUSSION THREAD

**Discussion by mimimix** [zepp-health/zeppos-forum#258] [User Feedback]
When you receive multiple notifications on your phone at the same time, the watch vibrates twice almost at once.
Setting the minimum time between vibrations would be very useful. (5, 10, 20, 30... sec).

**Comment by MyoungXUE** [comment]
Hi, this is a very good feature suggestion, I've already fed it back to the relevant colleagues and will consider adding it in subsequent OTAs!

**Reply by mimimix** [reply]
And you can also add an option to the notify method that will disable the notification vibration

**Reply by mimimix** [reply]
*Still waiting
_with gts4_

**Reply by mimimix** [reply]
Update 5.0.3.3 has appeared, but there is no such function :(
